.github/workflows/installMediaWiki.sh

```
@@ -0,0 +1,53 @@
+
+tar -zxf $MW_BRANCH.tar.gz
+mv mediawiki-$MW_BRANCH mediawiki
+
+cd mediawiki
+
+composer install
+if [ "$DB" = "mysql" ]; then
+    php maintenance/install.php --dbtype mysql --dbserver 127.0.0.1:3306 --dbuser mediawiki --dbpass mediawiki \
+        --dbname mediawiki --pass AdminPassword WikiName AdminUser
+else
+    php maintenance/install.php --dbtype sqlite --dbuser root --dbname mw --dbpath $(pwd) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)